stories/core_components/GeoPushpinChart.tsx

```diff
@@ -0,0 +1,101 @@
+// (C) 2019-2020 GoodData Corporation
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { VisualizationObject, VisualizationInput } from "@gooddata/typings";
+import { screenshotWrap } from "@gooddata/test-storybook";
+
+import { IGeoConfig } from "../../src/interfaces/GeoChart";
+import { GeoPushpinChart } from "../../src";
+import { onErrorHandler } from "../mocks";
+import {
+    ATTRIBUTE_LOCATION_GEOCHART,
+    MEASURE_SIZE_GEOCHART,
+    MEASURE_COLOR_GEOCHART,
+    ATTRIBUTE_SEGMENT_GEOCHART,
+    ATTRIBUTE_TOOLTIP_GEOCHART,
+} from "../data/geoChartComponentProps";
+
+const wrapperStyle = { width: 800, height: 500 };
+
+function renderGeoPushpinChart(
+    location: VisualizationObject.IVisualizationAttribute,
+    size: VisualizationObject.IMeasure,
+    color: VisualizationObject.IMeasure,
+    segmentBy: VisualizationObject.IVisualizationAttribute,
+    config: IGeoConfig,
+    filters: VisualizationInput.IFilter[],
```

Comment:
Please convert these parameters to object, so we can remove `null` parameters in function calls later